### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by demaceo**
#### What does  this PR do?
- Installs [ReactPlayer](https://www.npmjs.com/package/react-player), Redux, [@reduxjs/toolkit](https://redux-toolkit.js.org/), @testing-library/, & react-router-dom.
- Provides a general foundation for the Home page,
- Improves responsive design- specifically for the **NavBar**, **Banner**, **Body**,  **SoundBar**,  **Trio**, & **Announcement** components.
- Redirects page onClick of **Login** link in **NavBar** component.
#### How should this be manually tested?
In terminal:
1.  `cd cappua`
2. `git fetch`
2. `git checkout setup`
2. `npm start`

#### Any background context you want to provide?
- The src images are currently being randomly loaded & have no significance to the actual site.
- The code for the **NavBar** component could be cleaned up, specifically for how the `nav-container` and the `menu-container` are both being rendered.
- Trio component has conditional rendering that seems sloppy.
- Redux still needs to be incorporated.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,57 +1,36 @@
...
+      <div className="App">
+        <NavBar />
+        <Route
+          exact
+          path="/"
+          render={() => (
+            <>
+              <Banner />
+              <Body />
+              <SoundBar />
```
Comment (coopterrones) [inline src/App.js:25]: looks good.. did you sleep???

File: src/features/NavBar/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  return (
+    <div className="navbar-container">
+      <Link to="/">
+        <img
+          className="logo"
+          src="https://media.giphy.com/media/FxfVpjpQb1kwKadK9L/giphy.gif"
+          alt="logo"
+        />
+      </Link>
+      <div className="navbar-contents">
```
Comment (coopterrones) [inline src/features/NavBar/NavBar.js:28]: noice.. matches comp fine. are these working links now ? I know we dont have the pages but curious.. Ill play around with them

**Review by coopterrones — APPROVED** [review]

File: src/features/NavBar/NavBar.js (hunk 2 above)
Comment (demaceo) [reply src/features/NavBar/NavBar.js:28]: The **Login** link works! The other links do successfully redirect the page but there isn't yet a component made/rendered to display for them.

File: src/App.js (hunk 1 above)
Comment (demaceo) [reply src/App.js:25]: a weeee bit

**Review by ETBassist — APPROVED** [review]
Yo, this looks _awesome_

**Pull request merged**